ellis.yaml

```diff
@@ -108,6 +112,7 @@ resources:
             __dn_range_length__: { get_param: dn_range_length }
             __dns_ip__: { get_param: dns_ip }
             __dnssec_key__: { get_param: dnssec_key }
+            __etcd_ip__ : { get_param: etcd_ip }
```

Comment:
Surely this should be the `local_ip` if `etcd_ip` isn't specified?  Maybe I'm missing something...